File: Baekjoon_Online_Judge/NO_14659/14659.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 07                 | #
# | Day 14                   | #
# | 14659 한조서열정리하고옴ㅋㅋ | #
# ---------------------------- #


def run():
    N = int(r_input())
    max_height = 0
    result = 0
    cnt = 0

    for height in map(int, r_input().split()):
        if height > max_height:
            max_height = height
            result = max(result, cnt)
            cnt = 0
        
        else:
            cnt += 1
    
    print(max(result, cnt))


if __name__ == "__main__":
    run()

File: Baekjoon_Online_Judge/NO_14659/14659.c
#include <stdio.h>
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 07                 | 
 | Day 14                   | 
 | 14659 한조서열정리하고옴ㅋㅋ | 
 ---------------------------- 
*/

int main(){
    int N, max_height = 0, cnt = 0, result = 0;

    scanf("%d", &N);

    for(int i = 0; i < N; i ++){
        int height;

        scanf("%d", &height);

        if(height > max_height){
            max_height = height;
            result = result < cnt ? cnt : result;
            cnt = 0;
        }
        else cnt ++;
    }

    printf("%d", result < cnt ? cnt : result);
    return 0;
}
